server/routerlicious/packages/services-shared/src/socketIoServer.ts

```diff
@@ -260,6 +313,58 @@ class SocketIoServer implements core.IWebSocketServer {
 			this.redisClientConnectionManagerForPub.getRedisClient().quit(),
 			this.redisClientConnectionManagerForSub.getRedisClient().quit(),
 		]);
+		if (this.socketIoConfig?.pingPongLatencyTrackingEnabled) {
+			clearInterval(this.pingPongLatencyInterval);
+			this.flushPingPongLatencyTracking();
+		}
+	}
+
+	private initPingPongLatencyTracking(socket: Socket) {
+		if (!this.socketIoConfig?.pingPongLatencyTrackingEnabled) {
+			return;
+		}
+
+		let lastPingStartTime: number | undefined;
+		const packetCreateHandler = (packet: any) => {
+			if (packet.type === "ping") {
+				lastPingStartTime = Date.now();
```

Comment:
Nit, I think I read somewhere that Date.now is not good as `performance` (or something similar that our lumberjack use to merge duration)? Maybe use that?